pkg/ccl/streamingccl/replicationtestutils/testutils.go

```diff
@@ -106,21 +103,26 @@ type TenantStreamingClusters struct {
 	DestTenantSQL  *sqlutils.SQLRunner
 }
 
-// CreateDestTenantSQL creates a dest tenant SQL runner and returns a cleanup
+// StartDestTenant starts the destination tenant and returns a cleanup
 // function that shuts tenant SQL instance and closes all sessions.
 // This function will fail the test if ran prior to the Replication stream
 // closing as the tenant will not yet be active
-func (c *TenantStreamingClusters) CreateDestTenantSQL(ctx context.Context) func() error {
-	testTenant, destTenantConn := serverutils.StartTenant(c.T, c.DestSysServer,
-		base.TestTenantArgs{TenantID: c.Args.DestTenantID, DisableCreateTenant: true, SkipTenantCheck: true})
+func (c *TenantStreamingClusters) StartDestTenant(ctx context.Context) func() error {
+	c.DestSysSQL.Exec(c.T, `ALTER TENANT $1 START SERVICE SHARED`, c.Args.DestTenantName)
```

Comment:
I wonder if it is in our interest to also grant all capabilities to the tenant here.

Reply:
I'm inclined to keep this as is for now. I took a look at our runbook(s) and our roachtests and we currently don't do any post cutover granting. I can add something in a follow up PR once we add something to the runbook around this.

Reply:
I'm fine doing it later.

The runbook doesn't have the grants because it uses a configuration profile that creates a tenant with all capabilities by default.  The roachtests do grant the exmpt_from_rate_limiting capability IIRC.